tests/test_model_v2.py

```diff
@@ -1462,4 +1471,4 @@ def test_invalid_xml(xml_builder_factory):
 
     with pytest.raises(PyODataParserError) as e_info:
         MetadataBuilder(xml).build()
-    assert str(e_info.value) == 'Metadata document syntax error'
\ No newline at end of file
+    assert str(e_info.value) == 'Metadata document syntax error'
```

Comment:
Perhaps unnecessary change - editor defaults?

Reply:
That's good. Keep the change. Editors not adding new line at the end of file are not considered OK by me.